Description: Update for fields added in newer versions of process
Author: Ilias Tsitsimpis <i.tsitsimpis@gmail.com>
Bug-Debian: https://bugs.debian.org/847158
Forwarded: https://github.com/jgoerzen/hsh/pull/16

Index: b/HSH/Command.hs
===================================================================
--- a/HSH/Command.hs
+++ b/HSH/Command.hs
@@ -301,6 +301,15 @@ genericCommand c environ (ChanHandle ih)
 #if MIN_VERSION_process(1,2,0)
 			    , delegate_ctlc = False
 #endif
+#if MIN_VERSION_process(1,3,0)
+          , detach_console = False
+          , create_new_console = False
+          , new_session = False
+#endif
+#if MIN_VERSION_process(1,4,0)
+          , child_group = Nothing
+          , child_user = Nothing
+#endif
 			   }
     in do (_, oh', _, ph) <- createProcess cp
           let oh = fromJust oh'
@@ -320,6 +329,15 @@ genericCommand cspec environ ichan =
 #if MIN_VERSION_process(1,2,0)
 			    , delegate_ctlc = False
 #endif
+#if MIN_VERSION_process(1,3,0)
+          , detach_console = False
+          , create_new_console = False
+          , new_session = False
+#endif
+#if MIN_VERSION_process(1,4,0)
+          , child_group = Nothing
+          , child_user = Nothing
+#endif
 			   }
     in do (ih', oh', _, ph) <- createProcess cp
           let ih = fromJust ih'
